CredentialProvider.Microsoft/CredentialProviders/Vsts/BearerTokenProviders.cs

```diff
@@ -70,17 +70,20 @@ public bool ShouldRun(bool isRetry, bool isNonInteractive, bool canShowDialog)
     public class UserInterfaceBearerTokenProvider : IBearerTokenProvider
     {
         private readonly IAdalTokenProvider adalTokenProvider;
+        private readonly ILogger logger;
 
-        public UserInterfaceBearerTokenProvider(IAdalTokenProvider adalTokenProvider)
+        public UserInterfaceBearerTokenProvider(IAdalTokenProvider adalTokenProvider, ILogger logger)
         {
-            this.adalTokenProvider = adalTokenProvider;
+            this.adalTokenProvider = adalTokenProvider;
+            this.logger = logger;
         }
 
         public bool Interactive { get; } = true;
         public string Name { get; } = "ADAL UI";
 
         public async Task<string> GetTokenAsync(Uri uri, CancellationToken cancellationToken)
         {
+            logger.Minimal(string.Format(Resources.UIFlowStarted, this.Name));
```

Comment:
Would it be useful to have the URI in the log too? Nah probably not necessary

Reply:
I think the main point is to know that a pop-up is occurring rather than know the specific instance of it. I can add that in the logging just so we won't have to do it later.